indico/queries/workflow.py

```diff
@@ -134,23 +133,27 @@ def requests(self):
 class _WorkflowSubmission(GraphQLRequest):
 
     query = """
-        mutation workflowSubmissionMutation($workflowId: Int!, ${arg_}: {type_}, $recordSubmission: Boolean) {{
-            {mutation_name}(workflowId: $workflowId, {arg_}: ${arg_}, recordSubmission: $recordSubmission) {{
+        mutation workflowSubmissionMutation($workflowId: Int!, ${arg_}: {type_}, $recordSubmission: Boolean, $bundle: Boolean, $resultVersion: SubmissionResultVersion) {{
+            {mutation_name}(workflowId: $workflowId, {arg_}: ${arg_}, recordSubmission: $recordSubmission, bundle: $bundle, resultVersion: $resultVersion) {{
```

Comment:
If someone on an older IPA version runs this query and `resultVersion` is not on the schema this will break. How are you guys thinking of mitigating this?

Reply:
This is a super good point. Elias, here is an example of how we dealt with exposing an option to clients without expecting that the underlying server might not support it (in this case, predictOptions was the issue): https://github.com/IndicoDataSolutions/indico-client-python/blob/master/indico/queries/model_groups/model_groups.py#L291
You can see, Jerry had to basically format the graphene query string based upon user input - so "bundles" as an arg only gets used if "bundles" kwarg is specified by a user

Reply:
This is only a problem because customers might be running their own platform right? This wouldn't be a problem we were in control

Reply:
Even if we run the platform, the customer may be on an older version because the customer or us hasn't schedule an upgrade

Reply:
What would a case where we require a the clients be used with certain ipa version ranges? Having the clients support all ipa versions doesn't make much sense to me

Reply:
We don't right now to my knowledge have explicit pinning of what IPA platform versions a given client version supports or how many versions back a new client version should support but these are good points. I know it's been discussed but I don't think a final solution has been settled on. IMO it's hard to enforce this until we have automated testing set up for clients which we don't yet. And also platforms versions for them to run against. But Liz and Nic would better in terms of what policies we'll drive for and what constraints are preventing us from having created or enforcing them yet